src/ray/raylet/raylet.cc

```diff
@@ -154,6 +162,31 @@ void Raylet::HandleAccept(const boost::system::error_code &error) {
   DoAccept();
 }
 
+void Raylet::DoAcceptEventSocket() {
+  event_socket_acceptor_.async_accept(socket_,
+                                      boost::bind(&Raylet::HandleAcceptEventSocket, this,
+                                                  boost::asio::placeholders::error));
+}
+
+void Raylet::HandleAcceptEventSocket(const boost::system::error_code &error) {
+  if (!error) {
+    // TODO: typedef these handlers.
```

Comment:
add your name to the TODO, ut actually it seems ok without typedefs

Reply:
I copied this `TODO` from the previous code. I also agree that without typedefs is ok.